### PR TITLE
DDF-2359 fix issues ingesting images on a headless server

### PR DESCRIPTION
**Pull request by ryeats**
#### What does this PR do?
fix issue ingesting images caused by generating thumbnails on headless server
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@tbatie @adimka @bcwaters 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@shaundmorris
#### How should this be tested?
run a ddf on a headless server and successfully ingest an image and verify that a thumbnail is generated
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by bcwaters** [comment]
👍

File: distribution/ddf-common/src/main/resources/etc/system.properties [hunk 1]
```diff
@@ -31,6 +31,9 @@ javax.net.ssl.trustStorePassword=changeit
 javax.net.ssl.keyStoreType=jks
 javax.net.ssl.trustStoreType=jks
 
+#force java to run in headless mode for when the server doesn't have a display device
+java.awt.headless=true
```
Comment (coyotesqrl) [inline distribution/ddf-common/src/main/resources/etc/system.properties]: What if the server does have a display device? Does this impact a standard distribution and its ability to interact with the display? I assume not, as we're mediating everything through the browser?

Comment (ryeats) [inline distribution/ddf-common/src/main/resources/etc/system.properties]: Correct everything runs fine headless I was able to run on my on machine successfully and on a docker instance that did not have a display device and successfully ingest a jpg that generated a thumbnail.

**Comment by coyotesqrl** [comment]
My question's been answered. 
👍

**Comment by ryeats** [comment]
I couldn't reproduce the failure case for this on a docker instance so waiting on more information to make sure I can reproduce it and that thumbnails and that this indeed fixes it.

**Comment by shaundmorris** [comment]
👍 , we are planning to backport this as well, correct?

**Comment by ryeats** [comment]
yeah, ill create a branch and cherry pick it when it gets merged.

**Pull request merged**